ext/reflection/php_reflection.stub.php

```diff
@@ -829,3 +829,24 @@ public function getCallable(): callable {}
 
     public function getTrace(int $options = DEBUG_BACKTRACE_PROVIDE_OBJECT): array {}
 }
+
+/**
+ * @strict-properties
+ * @not-serializable
+ */
+final class ReflectionConstant
```

Comment:
I can second what @arnaud-lb write previously: it would be nice to implement `Reflector` along with the usual `__toString()` method.